src/matchbox/core.cljc

```diff
@@ -506,6 +506,12 @@
 
 ;; onAuth and offAuth are not wrapped yet
 
+(defn onAuth [ref cb]
+  (.onAuth ref (wrap-auth-cb cb)))
+
+(defn offAuth [ref cb]
+  (.offAuth ref cb))
```

Comment:
Do you mean for offAuth? offAuth disables the onAuth callback. I dont know if you want to track onAuth callbacks in the registry and then have offAuth called for removing them. onAuth callback will be called on both login and logout. So the auth callback must handle an auth map and nil.